src/Components/AlbumCard/AlbumCard.css

```diff
@@ -0,0 +1,45 @@
+.card {
+  margin: 3%;
+  background-color: #2a0800;
+  color: #ccfbfe;
+  padding: 20px;
+  width: 280px;
+  height: 507px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-self: center;
+}
+ .cover {
+  /* width: 250px; */
+  height: 250px;
+ }
+
+.date {
+  margin: 0;
+}
+
+ .discogs-link {
+  background-color: #FFBF00;
```

Comment:
Because I declared the variables on my branch this would be a great place to replace with <code>var(--accent-color)</code>!